### ISSUE TITLE
Change CLI to have required arguments

### ISSUE THREAD

**Issue by afuetterer** [ddionrails/collect_stata#27]
If I run collect_stata without an output directory, it still tries to do something and crashes.
```
$ collect_stata -i tests/input
...
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Maybe both -i and -o should be required to start the program?

Hint: You can declare arguments as `required=True`.

**Comment by afuetterer** [comment]
Does it make sense to run the CLI without --study?
Mabye -i, -o- and -s should be required?

**Comment by afuetterer** [comment]
I would argue that running the CLI without --study is useless.
https://github.com/ddionrails/collect_stata/blob/54f33d3d11e0b4e6ba1c4f989c56e293258c9b82/collect_stata/__main__.py#L32

The resulting JSON files contain `"study": ""`.
This in fact passes the JSON schema, but it will throw an error while importing into ddionrails.

I will update the schema, like this
https://stackoverflow.com/questions/38478406/ensure-one-property-is-not-empty-in-json-schema

Please make study required.

tldr:
- input
- output
- study
should be required arguments. @hansendx do you agree?
